src/ogd/games/THERMOLAB/features/AnswerAttemptsCount.py

```diff
@@ -0,0 +1,31 @@
+from typing import Any, Dict
+from ogd.core.generators.Generator import GeneratorParameters
+from ogd.core.generators.extractors.SessionFeature import SessionFeature
+from ogd.common.models.Event import Event
+from ogd.common.models.enums.ExtractionMode import ExtractionMode
+
+class AnswerAttemptsCount(SessionFeature):
+
+    def __init__(self, params: GeneratorParameters):
+        self.quiz_attempts: Dict[str, int] = {}
+        self.correct_answers: Dict[str, bool] = {}
+        super().__init__(params=params)
+
+    @classmethod
+    def _eventFilter(cls, mode: ExtractionMode) -> List[str]:
+        return ["click_select_answer", "click_submit_answer"]
+
+    def _updateFromEvent(self, event: Event) -> None:
```

Comment:
Players are able to freely select answers without submitting, and submitting an answer doesn't necessarily mean it's correct. So really, these should be named something like "answers_selected" and "answers_submitted", or similar.

Additionally, we should use subfeatures here, not a dictionary.